scripts/validate_docstrings.py

```diff
@@ -180,6 +185,11 @@ def deprecated(self):
                 bool(pattern.search(self.summary)) or
                 bool(pattern.search(self.extended_summary)))
 
+    @property
+    def mentioned_private_classes(self):
+        private_classes = ['NDFrame', 'IndexOpsMixin']
```

Comment:
Personally I'd keep this in a "constant" at the beginning of the file, but I it's just a personal preference.

Reply:
will do